gocd/announcer.gocd.yaml

```diff
@@ -36,3 +36,21 @@ pipelines:
           - staging-bot
         tasks:
           - script: ./factory-package-news/announcer.py --config config/announcer/openSUSE:Kubic.yml --verbose --state-file /home/go/config/announcer.yaml
+  Announcer.MicroOS:
+    group: openSUSE.Checkers
+    lock_behavior: unlockWhenFinished
+    environment_variables:
+      OSC_CONFIG: /home/go/config/oscrc-staging-bot
+    materials:
+      script:
+        git: https://github.com/openSUSE/openSUSE-release-tools.git
+    timer:
+      spec: 0 0 * ? * *
```

Comment:
please run at a different time (also for kubic) - as the announcers access the same state file, we shouldn't risk too much overwriting the file. I can also copy away the state file 3 times to erase this problem.

Reply:
Done, what do you think?